php/pantheon/checks/general.php

```diff
@@ -180,4 +181,47 @@ public function message(Messenger $messenger) {
 		}
 		$messenger->addMessage(get_object_vars($this));
 	}
+
+	public function checkCoreUpdates() {
+
+		ob_start();
+		\WP_CLI::run_command(array(
+			'core',
+			'check-update'
+		), array('format' => 'json'));
+		$ret = ob_get_clean();
```

Comment:
@danielbachhuber, ah now I remember why I kept the `ob_` functions which are also used in https://github.com/wp-cli/doctor-command/blob/master/inc/checks/class-core-update.php#L13

Without those, the json itself is printed to the output. I don't we want that printed.